.tool-versions

```diff
@@ -3,14 +3,13 @@
 # Be EXTREMELY CAREFUL with this. If you override a standard version
 # you are reducing compatibility guarantees.
 ## <<Stencil::Block(toolverOverride)>>
-## <</Stencil::Block>>
-golang 1.22.0
+golang 1.23.0
 nodejs 20.12.2
-protoc 21.5
-terraform 1.5.7
+## <</Stencil::Block>>
 # Note: Versions in this block do not override the default versions above
 # but sometimes you have to declare additional versions of the same tool
 # while leaving the 'default' version intact for the infra.
 # The most common case is nodejs.
 ## <<Stencil::Block(toolver)>>
+mage 1.14.0
```

Comment:
I'm generally not inclined to add this because it's already here:

https://github.com/getoutreach/devbase/blob/1ff9c81010a7e7771d29971daf45dde17f9eaa8c/.tool-versions#L16